### PR TITLE
аdd files to create a database and test data

### PR DESCRIPTION
**Pull request by andron13**
аdd files to create a database and test data
https://github.com/andron13/RecipeBackend/issues/24

### CODE REVIEW

File: DB-and-API/db-sql-create.sql [hunk 1]
```diff
@@ -1,55 +1,148 @@
...
+insert into user(user_loginname, user_email, user_password)
+values ('andron13', 'andron13@gmail.com', '123456'),
+       ('Juri', 'test1@test.dev', '123456'),
+       ('Elena', 'test2@test.dev', '123456'),
+       ('Dmitri', 'test3@test.dev', '123456');
+
+/* add 1 photo */
+
+insert into photo(photo_path, photo_title, photo_alt)
+values ('https://ru.inshaker.com/uploads/cocktail/hires/698/1537730384-Parkside-Fizz__highres.jpg',
```
Comment (sergeylukichev) [inline DB-and-API/db-sql-create.sql]: Is inshaker your web site? I would recommend to stick to something which you own, either some free hosting or aws s3 bucket, etc.

File: DB-and-API/db-sql-create.sql [hunk 2]
```diff
@@ -1,55 +1,148 @@
...
+
+/* add user_info */
+
+insert into user_info
+values (1, 'Андрей', 'Подлубный');
+
+/* add ingredient */
+
+insert into ingredient (ingredient_title, description, fk_ingredient_photo_id)
+values ('Водка', 'Водка, один из самых популярных ингредиентов для коктейлей', 3),
```
Comment (sergeylukichev) [inline DB-and-API/db-sql-create.sql]: do all ingredients look the same (ingredient_photo_id is 3 for all in your table)?

**Comment by sergeylukichev** [comment]
Have you tried to start the project with this schema so that it runs and works?

**Comment by sergeylukichev** [comment]
:+1:

File: DB-and-API/db-sql-create.sql (hunk 1 above)
Comment (andron13) [reply DB-and-API/db-sql-create.sql]: iI'm gonna do what you're suggest me to do this week.

**Pull request merged**